test/e2e/main/main_test.go

```diff
@@ -168,13 +165,19 @@ func TestNodeStartup(t *testing.T) {
 	spec.Annotations[v1.TargetLabels] = "my-svc-label"
 	spec.Labels = make(map[string]string)
 	spec.Labels["my-svc-label"] = "my-svc-value"
-	os.Setenv(v1.OperatorTargetLabelsEnvVarName, "{\"operator-svc-label\":\"operator-svc-value\"}")
-	os.Setenv(v1.OperatorTargetLabelsEnvVarName, "{\"operator-svc-label\":\"operator-svc-value\"}")
+	if err := os.Setenv(v1.OperatorTargetLabelsEnvVarName, "{\"operator-svc-label\":\"operator-svc-value\"}"); err != nil {
```

Comment:
`ExpectNoError` already checks if the passed param is nil, so we can just do: `ExpectNoError(os.Setenv(v1.OperatorTargetLabelsEnvVarName, "{\"operator-svc-label\":\"operator-svc-value\"}"))`

As this is test code and the test will fail if the env is not set anyway, I don't think checking err is really important here.